ClothesApp/ClothesApp/Controllers/BrandsController.cs

```diff
@@ -0,0 +1,123 @@
+using AutoMapper;
+using ClothesApp.Data;
+using ClothesApp.Data.Dtos;
+using ClothesApp.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace ClothesApp.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class BrandsController : ControllerBase
+{
+    private readonly ClothesAppContext _dbContext;
+
+    private readonly IMapper _mapper;
+    
+    public BrandsController(ClothesAppContext dbContext, IMapper mapper)
+    {
+        _dbContext = dbContext;
+        _mapper = mapper;
+    }
+
+    [HttpGet]
+     public async Task<ActionResult<IReadOnlyCollection<BrandDto>>> GetAllBrands()
+     { 
+         var brands = await _dbContext.Brands.ToListAsync();
+         
+         var brandDtos = _mapper.Map<IReadOnlyCollection<BrandDto>>(brands);
+         
+         return Ok(brandDtos);
+     }
+    
+    [HttpGet("{id}")]
+    public async Task<ActionResult<BrandDto>> GetBrandById(long id)
+    {
+        if (id <= 0)
+        {
+            return BadRequest($"Brand Id = '{id}' can't be less or equal zero");
+        }
+        
+        var brand = await _dbContext.Brands.FindAsync(id);
+
+        if (brand is null)
+        {
+            return NotFound($"Brand with Id = '{id}' not found");
+        }
+        
+        var brandDto = _mapper.Map<BrandDto>(brand);
+        return Ok(brandDto);
+    }
+
+    [HttpDelete("{id}")]
+    public async Task<IActionResult> DeleteBrandById(long id)
+    {
+        if (id <= 0)
+        {
+            return BadRequest($"Brand Id = '{id}' can't be less or equal zero");
+        }
+        
+        var brand = await _dbContext.Brands.FindAsync(id);
+
+        if (brand is null)
+        {
+            return NotFound($"Brand with Id = '{id}' not found");
```

Comment:
You have similar messages in more than one place, consider moving all validation messages to a constants set